user/downloading-installing-upgrading/supported-versions.md

```diff
@@ -75,6 +73,45 @@ Currently, only [Fedora] and [Debian] TemplateVMs are officially supported by th
 extensive testing.
 
 
+## TemplateVM support status
+
+The tables below shows the support status of each [TemplateVM] provided by the Qubes OS Project.
+Currently, only [Fedora] and [Debian] TemplateVMs are officially supported by the Qubes OS Project.
+[Whonix] TemplateVMs are supported by our partner, the [Whonix Project].
+
+
+### Fedora
+
+Qubes support for each [Fedora] TemplateVM ends when that Fedora release reaches [end-of-life][fedora-eol], unless otherwise noted.
+
+| TemplateVM    | Status                  |
+| ------------- | ----------------------- |
+| Fedora 18     | Old, unsupported        |
+| Fedora 20     | Old, unsupported        |
+| Fedora 21     | Old, unsupported        |
+| Fedora 22     | Old, unsupported        |
+| Fedora 23     | Old, unsupported        |
+| Fedora 24     | Old, unsupported        |
+| Fedora 25     | Old, unsupported        |
+| Fedora 26     | Old, unsupported        |
+| Fedora 27     | Old, unsupported        |
+| Fedora 28     | Old, unsupported        |
+| Fedora 29     | Old, unsupported        |
+| Fedora 30     | Current, supported      |
+
+
+### Debian
+
+Qubes support for each [Debian] TemplateVM ends when that Debian release reaches [end-of-life][debian-eol], unless otherwise noted.
+
+| TemplateVM    | Status                  |
+| ------------- | ----------------------- |
+| Debian 7      | Old, unsupported        |
+| Debian 8      | Old, unsupported        |
```

Comment:
@marmarek, IIRC, you started a discussion about dropping support for Debian 8 on qubes-devel. I don't recall seeing a resolution to that question. Did you decide to drop support? If so, how should we represent that here?

Reply:
> Did you decide to drop support?

Yes.

> If so, how should we represent that here?

Something like this. TBH I don't understand what "old"/"current" means in this table - is it just another word for "unsupported"/"supported"?

Reply:
>  TBH I don't understand what "old"/"current" means in this table - is it just another word for "unsupported"/"supported"?

"Old" means that it is not the newest release. An old release can still be supported. For example, when Qubes 4.0 came out, 3.2 was still supported, but it was the old version.

However, we can drop "old"/"new" in order to simplify the table.